app/frontend/wallet/constants.ts

```diff
@@ -22,6 +22,18 @@ export const NETWORKS = {
     },
   },
   SHELLEY: {
+    MAINNET: {
+      name: 'shelley-mainnet',
+      networkId: 0,
+      protocolMagic: 764824073,
+      ttl: undefined,
+    },
+    HASKELL_TESTNET: {
+      name: 'htn',
+      networkId: 0,
+      protocolMagic: 764824073,
```

Comment:
this should be 42 indeed

Reply:
can be seen e.g. here: https://iohk.zendesk.com/hc/en-us/articles/900001220783-Transaction-errors-Failed-reading-Your-input-is-either-malformed-or-not-hex-encoded